kong/pdk/response.lua

```diff
@@ -244,7 +244,7 @@ local function new(self, major_version)
       error("header name must be a string", 2)
     end
 
-    local header_value = _RESPONSE.get_headers()[name]
+    local header_value = ngx.header[name]
```

Comment:
Should we keep the original code here?

Reply:
`ngx.header` is the better way to get a single response header, and it returns the same type of values as `get_headers()[name]`: if `get_headers()[name]` get a table, `ngx.header` will get a table similarly(unlike `ngx.var.sent_http_` which returns comma separated string). So we make this change for better performance.